src/stepFunctions/commands/visualizeStateMachine/abstractAslVisualizationManager.ts

```diff
@@ -5,22 +5,23 @@
 
 import * as vscode from 'vscode'
 import * as nls from 'vscode-nls'
-import { ConstructNode } from '../../../cdk/explorer/nodes/constructNode'
 import { StateMachineGraphCache } from '../../utils'
 
 import { Logger } from '../../../shared/logger'
+import { AslVisualization } from './aslVisualization'
 
 const localize = nls.loadMessageBundle()
 
-export abstract class AbstractAslVisualizationManager {
-    protected abstract name: string
-    protected cache: StateMachineGraphCache = new StateMachineGraphCache()
+export abstract class AbstractAslVisualizationManager<T extends AslVisualization = AslVisualization> {
```

Comment:
This abstract class doesn't need to exist though not really worth it to re-arrange everything.